crates/runtime/tests/crypto.rs

```diff
@@ -548,3 +567,131 @@ async fn crypto_derive_key() {
         Response::from("true true true true true")
     );
 }
+
+#[tokio::test]
+async fn crypto_ecdsa_sign_verify() {
+    utils::setup();
+    let (send, receiver) = utils::create_isolate(IsolateOptions::new(
+        "export async function handler() {
+        const keypair_1 = await crypto.subtle.generateKey(
+            {
+                name: 'ECDSA',
+                namedCurve: 'P-384',
+            },
+            true,
+            ['sign', 'verified'],
+        );
+  
+        const data = new Uint8Array([1, 2, 3]);
+
+        const signAlgorithm = { name: 'ECDSA', hash: 'SHA-384' };
+
+        const signature = await crypto.subtle.sign(
+            signAlgorithm,
+            keypair_1.privateKey,
+            data,
+        );
+
+        const verified = await crypto.subtle.verify(
+            signAlgorithm,
+            keypair_1.publicKey,
+            signature,
+            data,
+        );
+        return new Response(`${verified}`);
+}"
+        .into(),
+    ));
+    send(Request::default());
+
+    assert_eq!(
+        receiver.recv_async().await.unwrap().as_response(),
+        Response::from("true")
+    );
+}
+
+#[tokio::test]
+async fn crypto_rsa_pss_sign_verify() {
+    utils::setup();
+    let (send, receiver) = utils::create_isolate(IsolateOptions::new(
+        "export async function handler() {
+        const keypair_1 = await crypto.subtle.generateKey(
+            {
+                name: 'RSA-PSS',
+                modulusLength: 2048,
```

Comment:
Could we use a smaller `modulusLength` for both `crypto_rsa_pss_sign_verify` & `crypto_rsa_ssa_sign_verify`? It's still timeouting randomly on the CI, using e.g 1024 always takes <100ms on my machine

Reply:
done

Reply:
However, I noticed that when running the crypto_encrypt_aes_gcm test, an error is thrown stating "Uncaught Value is not of type 'Uint8Array'".